frontend/src/locales/en-US.json

```diff
@@ -845,7 +845,11 @@
       "no-earlier-than": "Restore point-in-time cannot be earlier than [{earliest}] (the earliest available backup).",
       "no-later-than-now": "Restore point-in-time cannot be later than now.",
       "restore-before-last-migration": "Restore to the point before last migration",
-      "restore-before-last-migration-help-info": "@:{'database.pitr.restore-before-last-migration'}. {link}."
+      "restore-before-last-migration-help-info": "@:{'database.pitr.restore-before-last-migration'}. {link}.",
+      "target": "Restore to",
+      "target-new-db": "New database",
+      "target-inplace": "In place",
```

Comment:
The key looks like not matches the value.

Reply:
SGTM